src/status_im/ui/screens/accounts/login/models.cljs

```diff
@@ -113,19 +126,31 @@
 
                       :else
                       wrap-with-stop-node-fx)]
-    (wrap-fn db' address password)))
+    (wrap-fn db' address password save-password)))
 
-(defn login-handler [login-result address {db :db}]
+(defn login-handler [login-result address password save-password {db :db}]
   (let [data    (types/json->clj login-result)
         error   (:error data)
         success (zero? (count error))
         db'     (assoc-in db [:accounts/login :processing] false)]
     (if success
-      {:db             db
-       :clear-web-data nil
-       :change-account [address]}
+      (merge
+       {:db             db
+        :clear-web-data nil
+        :change-account [address]}
+       (when save-password
+         {:dispatch [:save-password address password]}))
```

Comment:
Exactly, when the `:save-password` event does nothing else other then produces `::save-password` effect, there is no reason for it to be an event (unless some outside action in app requires just the `::save-password` effect to happen and nothing more, which is not the case).
It would be better to implement only `:save-password` effect (global and not namespaces keyword as `::save-password`) and no event, then this effect could be included in return value of any handler function where it's needed, with no unnecessary event chain.